mxcube3/routes/Login.py

```diff
@@ -115,7 +115,8 @@ def signout():
     mxcube.queue = qutils.new_queue()
     mxcube.shapes.clear_all()
     qutils.reset_queue_settings()
-    mxcube.session.clear_session()
+    if hasattr(mxcube.session, 'clear_session'):
```

Comment:
Why do we need this check for session in particular and not for all HO's ?, it seems to be like there is a configuration problem if you need to do this check. If that's the case we better check for shapes as well ...

Reply:
I have not submit those lines for the hardware repo, so I added this here in the meantime